contracts/universalSchemes/GlobalConstraintRegistrar.sol

```diff
@@ -35,16 +35,11 @@ contract GlobalConstraintRegistrar is UniversalScheme {
         bytes32 voteToRemoveParams; // Voting parameters for removing this GC.
     }
 
-    // Struct holding the data for each organization
-    struct Organization {
-        bytes32 voteRegisterParams; // The voting parameters for adding a GC.
-        IntVoteInterface intVote; // The voting machine in which the voting takes place.
-        mapping(bytes32=>GCProposal) proposals; // A mapping from the proposal ID to the proposal itself.
-        mapping(address=>bytes32) voteToRemoveParams; // A mapping that saves the parameters for removing each GC.
-    }
+    // GCProposal by avatar and proposalId
+    mapping(address=>mapping(bytes32=>GCProposal)) public organizationProposals;
 
-    // A mapping from the organization (Avatar) address to the saved data of the organization:
-    mapping(address=>Organization) public organizationsData;
+    // voteToRemoveParams hash by avatar and proposal.gc
+    mapping(address=>mapping(address=>bytes32)) public voteToRemoveParams;
```

Comment:
why do we need additional mapping here?
voteToRemoveParams could be part of  organizationsProposals .

Reply:
Because those mappings are not accessible when contained inside the struct.  See #446.

Reply:
Maybe you are suggesting moving the `voteToRemoveParams` _values_ (not the mapping) into the `GCProposal` struct?  That field actually is already there.  But the `voteToRemoveParams`  mapping is indexed by the GC address, unlike the `organizationProposals` mapping which is indexed by `proposalId`.  Such a mapping is needed by `proposeToRemoveGC`.

Reply:
How about just provide a getter function to make these mappings accessible ?

Reply:
A function would not be consistent with the other schemes that create proposals that all have a mapping named `organizationsProposals`.

In any case, how would  a function be advantageous?

If the `voteToRemoveParams` mapping is not desired, then perhaps it doesn't need to be public.

Reply:
BTW, from the point of view of javascript, `organizationProposals` is already a function.

Reply:
Done